e2e/testcases/stress_test.go

```diff
@@ -846,37 +847,60 @@ func validateNumberOfObjectsEquals(nt *nomostest.NT, gvk schema.GroupVersionKind
 	return nil
 }
 
-func pauseDeploymentYAML(name, namespace string) string {
-	return fmt.Sprintf(`
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: %s
-  namespace: %s
-  labels:
-    app: %s
-    nomos-test: enabled
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: %s
-  template:
-    metadata:
-      labels:
-        app: %s
-    spec:
-      containers:
-      - name: pause
-        image: registry.k8s.io/pause:3.7
-        ports:
-        - containerPort: 80
-        resources:
-          limits:
-            cpu: 250m
-            memory: 256Mi
-          requests:
-            cpu: 250m
-            memory: 256Mi
-`, name, namespace, name, name, name)
+func pauseDeploymentObject(nt *nomostest.NT, name, namespace string) *appsv1.Deployment {
+	labels := map[string]string{"app": name}
+	deployment := k8sobjects.DeploymentObject(
+		core.Name(name), core.Namespace(namespace), core.Labels(labels))
+	replicas := int32(1)
```

Comment:
The pointer library makes this a one liner.

Reply:
done